src/ClusterSequence.jl

```diff
@@ -378,8 +380,10 @@ n_exclusive_jets(clusterseq, dcut = 20.0)
 """
 function n_exclusive_jets(clusterseq::ClusterSequence; dcut::AbstractFloat)
     # Check that an algorithm was used that makes sense for exclusive jets
-    if !(clusterseq.algorithm ∈
-         (JetAlgorithm.CA, JetAlgorithm.Kt, JetAlgorithm.EEKt, JetAlgorithm.Durham))
+    if clusterseq.algorithm == JetAlgorithm.GenKt && clusterseq.power < 0
+        throw(ArgumentError("Algorithm GenKt requires power >= 0 (power=$(clusterseq.power))"))
+    elseif clusterseq.algorithm ∉
+           (JetAlgorithm.CA, JetAlgorithm.Kt, JetAlgorithm.EEKt, JetAlgorithm.Durham)
```

Comment:
There is a logic flaw here! If you have GenKt and p >= 0, this test will *fail* and it should pass.

```sh
 ~/.julia/dev/JetReconstruction/examples/ [genkt_pp] julia --project jetreco.jl -p 0.5 -A GenKt -R 1.0 --exclusive-njets 2 ../test/data/events.pp13TeV.hepmc3.gz
[ Info: Total Events: 100
[ Info: Jet reconstruction will use GenKt with power 0.5
[ Info: Will produce exclusive jets with n_jets = 2
ERROR: LoadError: ArgumentError: Algorithm used is not suitable for exclusive jets (GenKt)
...
```